mubench.pipeline/tests/data/test_runner_interface.py

```diff
@@ -0,0 +1,96 @@
+import logging
+from collections import OrderedDict
+from os.path import join
+from tempfile import mkdtemp
+from unittest import mock
+from unittest.mock import MagicMock, PropertyMock, patch
+
+from nose.tools import assert_equals, assert_true, assert_raises
+
+from data.runner_interface import NoInterface, RunnerInterface, RunnerInterfaceV20170406, NoCompatibleRunnerInterface
+from tests.data.stub_detector import StubDetector
+from utils.io import remove_tree, write_yaml
+from utils.shell import Shell, CommandFailedError
+from tests.test_utils.data_util import create_misuse, create_version, create_project
+from tests.test_utils.runner_interface_test_impl import RunnerInterfaceTestImpl
+
+class TestRunnerInterface:
+    def test_get_interface_version(self):
+        actual = RunnerInterface.get(RunnerInterfaceTestImpl.TEST_VERSION, "", dict())
+        assert_true(isinstance(actual, RunnerInterfaceTestImpl))
+
+    def test_get_interface_version_default_for_none(self):
```

Comment:
If a release file is missing the cli_version, the detector defaults it to None. We could also let the detector raise an exception instead, as None should probably not be a valid parameter value for the factory method. Should I change that?